support-frontend/assets/pages/contributions-landing/components/SepaForm.tsx

```diff
@@ -17,36 +26,78 @@ const fieldsContainerStyles = css`
 		margin-top: ${space[3]}px;
 	}
 `;
+
 // -- Component -- //
 type SepaFormProps = {
 	iban: string | null;
 	accountHolderName: string | null;
+	addressStreetName?: string;
+	addressCountry?: Country;
+	updateAddressStreetName: (addressStreetName: string) => void;
+	updateAddressCountry: (addressCountry: Country) => void;
 	updateIban: (iban: string) => void;
 	updateAccountHolderName: (accountHolderName: string) => void;
 	checkoutFormHasBeenSubmitted: boolean;
 };
 export function SepaForm({
 	iban,
 	accountHolderName,
+	addressStreetName,
+	addressCountry,
+	updateAddressStreetName,
+	updateAddressCountry,
 	updateIban,
 	updateAccountHolderName,
 	checkoutFormHasBeenSubmitted,
 }: SepaFormProps) {
+	const [requireAddress, setRequireAddress] = useState(true);
+
+	const isAddressRequired = (_iban: string): boolean => {
+		// const requiredAddressPrefixes = [
+		// 	'AD',
+		// 	'PF',
+		// 	'TF',
+		// 	'GI',
+		// 	'GB',
+		// 	'GG',
+		// 	'IM',
+		// 	'JE',
+		// 	'MC',
+		// 	'NC',
+		// 	'BL',
+		// 	'PM',
+		// 	'SM',
+		// 	'CH',
+		// 	'WF',
+		// ];
+		//
+		// return requiredAddressPrefixes.includes(iban.substring(0, 2).toUpperCase());
+
+		return true;
+	};
```

Comment:
should we delete this for now?

Reply:
Could do. I just thought it was worth keeping so we could reinstate it easily. I'm easy either way.